test/e2e/schemas.js

```diff
@@ -26,8 +26,10 @@ exports.domain = {
             minItems: 1
         },
         name: { type: "string" },
+        created: { type: "string" },
+        published: { type: "string" },
```

Comment:
require datetime instead of string? (twice)

Reply:
done